index.js

```diff
@@ -7,6 +7,13 @@ var funnel = require("broccoli-funnel");
 var merge = require("broccoli-merge-trees");
 var path = require("path");
 
+function getDefaultAssetHttpPrefix(parent) {
+  if (parent && parent.options && parent.options.lazyLoading) {
```

Comment:
I think this should just `parent.lazyLoading` (it's supposed to be the instance ([ref](https://github.com/ember-engines/ember-engines/blob/aa2ec6a11316f44f3c832919d7a835de8f5492ae/lib/engine-addon.js#L266))).